docs/reference/services.rst

```diff
@@ -4,10 +4,64 @@
 MicroOVN services
 =================
 
+MicroOVN has lots of its functionality separated into distinct services that can
+be controlled via MicroOVN enable/disable or snap/systemd, only the former is
+recommended.
+
 This page presents a list of all MicroOVN services. Their descriptions are
 for reference only - the user is not expected to interact directly with these
 services.
 
+Handling services with enable/disable
+-------------------------------------
+
+The status of all services is displayed by running:
+
+.. code-block:: none
+
+   microovn status
+
+``central service``
+-------------------
+
+This is responsible for the database control. The database is a RAFT database so
```

Comment:
... database is clustered and uses the Raft algorithm for consensus.  It can ...